docs/swagger/swagger.yaml

```diff
@@ -4223,6 +4223,11 @@ paths:
         schema:
           default: 120s
           type: string
+      - description: 'Data filter field. Prefixes supported: > >= < <= @ ^ ! !@ !^'
```

Comment:
Is there somewhere we can indicate in the docs/swagger that this is a date/time? I had to read further into the source code to see where it gets converted to tell what type it actually was.

Reply:
Probably not. The Swagger spec still needs a lot of fleshing out in general... I've noted before that the Swagger generator could almost be a separate project in itself. But I filed https://github.com/hyperledger/firefly/issues/214 for this.